Client/game_sa/CHudSA.cpp

```diff
@@ -922,6 +928,27 @@ static void HOOK_RenderHudBar(int playerId, int x, int y)
         CHudSA::RenderArmorBar(x, y);
 }
 
+////////////////////////////////////////////////////////////////////////
+// CStats::GetFatAndMuscleModifier
+// 
+// After changing MAX_HEALTH from 176 to 200, we need to maintain the HP proportions
+// meaning stat = 569 = 100 HP, stat = 1000 = 200 HP
+//
+// HP = a * STAT - b
+// a = (100/431) = 0.23196
+// b = 32 (we use 31.9 due to rounding the result down - with 32, instead of 100 HP, it's 99)
+////////////////////////////////////////////////////////////////////////
+static const float mult = 0.23196f;
+static const float subVal = 31.9f;
+static void HOOK_MaxHealthStat_Correction()
+{
+    _asm
+    {
+        fmul dword ptr [mult] // stat * a
+        fsub dword ptr [subVal] // stat * a - b
+    }
+}
+
```

Comment:
Technically, it's a core gameplay mechanic.
Why can't we just increase hp bar width if a player has > 176 health?

Reply:
Answered below